cirq/contrib/quirk/cells/cell.py

```diff
@@ -89,6 +105,18 @@ def modify_column(self, column: List[Optional['Cell']]) -> None:
             Nothing. The `column` argument is mutated in place.
         """
 
+    def persistent_modifiers(self) -> Dict[str, Callable[['Cell'], 'Cell']]:
+        """Overridable modifications to apply to the rest of the circuit.
+
+        Persistent modifiers apply to all cells in the same column, not just to
+        future columns.
```

Comment:
This comment is confusing. Does a persistent modifier apply to future columns or the same column as `self` is in? Also, what about past columns?

Reply:
Rewritten.